src/pperm.c

```diff
@@ -6322,8 +6214,8 @@ Obj OnSetsPPerm(Obj set, Obj f)
                     *ptres++ = tmp;
                 }
             }
-            else { /* this case cannot occur since I think POW is not defined
-                      */
+            else {
+                // This case cannot occur since POW is not defined
```

Comment:
I don't understand this comment. Best I can come up with is that `POW` is a macro, and clearly defined as such... so clearly you mean something else, but what?

Reply:
This is an old comment, and I do not know what I meant either, my memory is that I couldn't get the code to enter this case. I will update this.

Reply:
Actually, I remember what this means now, after putting the analogous code from `permutat.c` in here instead of the error. I have updated the comment.